README.md

```diff
@@ -165,150 +162,126 @@ upper right corner.  ![](https://raw.githubusercontent.com/deephaven-examples/de
+To activate the virtual environment:
+```bash
+source ./venv-<versiondetails>/bin/activate
+```
 
-The pip-installed Deephaven uses a lightweight Deephaven installation that is installed using pip.  In this case,
-the pip-installed Deephaven system is installed directly on your local system, without Docker.  
-This is a good option for production scenarios where scripts need to be run and related data needs to be visualized.
+Once the virtual environment is activated, `python` and `pip` will use the virtual environment's Python and packages --
+including everything needed to run [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib).
 
-It is possible to use [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib) without docker, but this is a 
-new feature and has not been well tested.  To do this:
-1) Install `ibapi`:
-    ```bash
-    # pip installed version of ibapi is too old.  You must download and install a more recent version.
-    export IB_VERSION=1016.01
-    curl -o ./api.zip "https://interactivebrokers.github.io/downloads/twsapi_macunix.${IB_VERSION}.zip"
-    unzip api.zip
-    cd ./IBJts/source/pythonclient
-    python3 setup.py install
-    ```
-2) Install Java 11 and set the appropriate `JAVA_HOME` environment variable.    
-3) Install [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib):
-    ```bash
-    pip3 install --upgrade pip setuptools wheel
-    pip3 install deephaven-ib
-    ```
-4) Launch the system and execute a custom script:
-    ```bash
-    # your_script.py must begin with: "import os; from deephaven_server import Server; _server = Server(port=10000, jvm_args=['-Xmx4g','-Dauthentication.psk=DeephavenRocks!','-Dstorage.path=' + os.path.expanduser('~/.deephaven')]); _server.start()"
-    # Deephaven IDE configuration and notebooks are stored to ~/.deephaven
-    # Set jvm_args to the desired JVM memory for Deephaven
-    python3 -i your_script.py
-    ```
-5) Launch the [Deephaven IDE](https://github.com/deephaven/deephaven-core/blob/main/README.md#run-deephaven-ide) by navigating to [http://localhost:10000/ide/](http://localhost:10000/ide/) in a browser and log in with the password `DeephavenRocks!`.
-6) Use `host=localhost` for the hostname in the examples
+### Deactivate the Virtual Environment
+
+To deactivate the virtual environment:
+```bash
+deactivate
+```
+
+Once the virtual environment is deactivated, `python` and `pip` will use the system's Python and packages.
+[deephaven-ib](https://github.com/deephaven-examples/deephaven-ib) will not be available.
+
+
+# Use deephaven-ib
+
+## Start Python
+
+To start Python with the virtual environment, run:
+```bash
+source ./venv-<versiondetails>/bin/activate
+python
+```
+
+## Start Deephaven
 
-# Authentication
+To start Deephaven, run the following python script.  It can be found at [./examples/run_deephaven.py](./examples/run_deephaven.py).  
```

Comment:
This seems to suggest that I should have installed Deephaven through pip and not Docker. This doc does not make that clear early on.